### PR TITLE
Added regex responder in the General cog and added GCI response

### LINKED ISSUE #266: GCI Invite Link on mention of GCI
**Is your feature request related to a problem? Please describe.**
As we draw closer to the start of GCI, many CD users continue to question what GCI is and how they can get involved.

**Describe the solution you'd like**
On a mention of GCI, the bot should give a brief explanation of GCI and a link to the official CD linked GCI server. This should be on a channel cooldown to avoid the message being spammed on every mention of GCI. I would be willing to write this if you approved.

**Describe alternatives you've considered**
Similar to the FAQ part of the bot we could instead add the questions like 'What is GCI?' but this may not catch everything. 

**Additional context**
N/a

### PR DESCRIPTION
**Pull request by JoshHeng**
Fixes #266

### CODE REVIEW

File: cdbot/cogs/cyber.py [hunk 1]
```diff
@@ -51,7 +51,10 @@ class Cyber(Cog):
         (r"^.*\bwhat\b.*\belite\b.*\bemail\b.*$", "**Quote from the Cyber Discovery Elite team: **"
          "We’re currently allocating students to their preferred locations so it’s an ongoing process! "
          "We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
-         )
+         ),
+
+        #Other events
+        (r"^.*\bwhat\b.*\b((gci)|(google\b.*\bcode))\b.*$", "Google Code-in is a global, open source competition where students aged 13-17 contribute to open source projects and earn prizes. Find out more at https://codein.withgoogle.com/ or join the Discord server at 651156140796674098")
```
Comment (thebeanogamer) [inline cdbot/cogs/cyber.py]: `651156140796674098` wot

**Comment by JoshHeng** [comment]
Ok, I'll just leave it here then. As a sidenote, I still don't know how the build is failing because now I've literally just added a few extra lines in the same format as the rest.

File: cdbot/cogs/cyber.py (hunk 1 above)
Comment (JoshHeng) [reply cdbot/cogs/cyber.py]: I copied the Discord ID instead of the link

**Comment by thebeanogamer** [comment]
You can see why tests are failing [here](https://dev.azure.com/cyberdiscovery/Cyber%20Discovery%20Discord%20Bot/_build/results?buildId=534&view=ms.vss-test-web.build-test-results-tab)

**Comment by thebeanogamer** [comment]
Closed as stale

**Pull request closed without merging**